static/css/app.css

```diff
@@ -334,6 +334,10 @@ section,
 summary {
     display: block;
     padding-bottom: 0.5%;
+    padding-inline-start: 1.5%;
+    position:fixed;
```

Comment:
So any menu element or any other will be fixed to the bottom? I'd still like to see a separate change for footer elements only.